### DISCUSSION TITLE
Something About Eventing Government

### DISCUSSION THREAD

**Discussion by JasonChen86899** [OpenFunction/OpenFunction#489] [Ideas]
Hi members. Recently I designed an EDA system and want to start a discussion about Eventing Government.
I have reviewed code of related opensoure project such as knative eventing, argo event and numaflow. So I summarize a standard spec of Eventing
Here is my doc
https://docs.google.com/document/d/1IsKXpmij_ALQV-6nUFDCYW4ngEbQYMy2Py1WL_Ae2Wc/edit?usp=sharing

I want some suggestion from openfunction community. Thx

**Comment by benjaminhuo** [comment]
@JasonChen86899 We can discuss this in the community meeting Next week.
You can join wechat group here:
![image](https://github.com/OpenFunction/OpenFunction/assets/18525465/9ef60fd7-b44e-47d6-9dde-2f2772707e17)

**Reply by JasonChen86899** [reply]
okay
